### PR TITLE
Update so that margins auto by default and 170 or 270 classes apply regardless of screen size

### PR DESCRIPTION
**Pull request by alexhoffman617**
Fixes inset images to be correct size and centered on smaller screens

## Changes

- Updates to inset.less

## How to test this PR

1. Visit the following link while running locally: 
http://localhost:8000/start-small-save-up/start-saving/how-to-use-your-tax-refund-to-build-your-emergency-funds/
2. Using Chrome tools change the size dynamically to make sure the page images still look good.

## Checklist

- [X] PR has an informative and human-readable title
- [X] Changes are limited to a single goal (no scope creep)
- [X] Code follows the standards laid out in the [CFPB development guidelines](https://github.com/cfpb/development)
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

### CODE REVIEW

**Review by Scotchester — CHANGES_REQUESTED** [review]
New lines 93–95 need to be removed. They're now overriding the `width` set by `&__170` above.

**Comment by alexhoffman617** [comment]
Ok @Scotchester , those lines are removed.

File: cfgov/unprocessed/css/molecules/inset.less [hunk 1]
```diff
@@ -74,25 +76,25 @@
...
+        width: unit( 170px / @base-font-size-px, em );
+    }
+
+    &__270 {
+        width: unit( 270px / @base-font-size-px, em );
+    }
+
     .respond-to-min( @bp-sm-min, {
         float: right;
         width: unit( 270px / @base-font-size-px, em );
```
Comment (Scotchester) [inline cfgov/unprocessed/css/molecules/inset.less:89]: Oops, this one needs to go, too.

```suggestion
```

Comment (Scotchester) [reply cfgov/unprocessed/css/molecules/inset.less:89]: And then I would also move the `float: right;` into a `&__right` block in the mixin below. We don't need two `.respond-to-min( @bp-sm-min, {…} )` mixins right never to each other.

**Review by Scotchester — CHANGES_REQUESTED** [review]

File: cfgov/unprocessed/css/molecules/inset.less (hunk 1 above)
Comment (Scotchester) [reply cfgov/unprocessed/css/molecules/inset.less:89]: Wait, argh. Let me think about this.

Those defaults may be being relied upon by those other components that share the `.m-inset` base class. I'll play around a bit.

Comment (alexhoffman617) [reply cfgov/unprocessed/css/molecules/inset.less:89]: Is there anything I can do to help you with this. Looking at something, pairing, etc.

Comment (Scotchester) [reply cfgov/unprocessed/css/molecules/inset.less:93]: Paired with Alex to confirm that we do in fact need a default for some blocks that just use `.m-inset` and rely on the default width and floating of that. Pushed a commit to make the 170px inset image selector more specific.

**Review by Scotchester — APPROVED** [review]

**Pull request merged**